examples/data_parallel/train_resnet_ddp.py

```diff
@@ -8,13 +8,15 @@
 from torch.nn.parallel import DistributedDataParallel as DDP
 import torch.optim as optim
 import torch_xla.distributed.xla_multiprocessing as xmp
+import torch_xla
+import torch_xla.core.xla_model as xm
 
 
 class TrainResNetDDP(TrainResNetBase):
 
   def __init__(self):
-    super().__init__()
     dist.init_process_group('xla', init_method='xla://')
+    super().__init__()
```

Comment:
nit: should the super init stay at the top?

Reply:
No, super().__init__() will call some functions from runtime, which needs init the TPU backend first.